### PR TITLE
[BO] Fix time-val input buttons to change calendar date-range

### LINKED ISSUE #25966: [BO] Dashboard time-val selection doesn't change the calendar
Hi there,

Selecting a time-val (day, month, year,...) from the calendar form, doesn't change the date-picker and causes a jquery error.

1. Go to BO > Dashboard
2. Select a time-val from calendar form 

image

3. See error in browser console

**Screenshots**

Screenshot 2021-09-22 at 12-11-40 پیشخوان • ONE

Screenshot 2021-09-22 121231

#### Additional information

* PrestaShop version: 1.7.8.0 (RTL)
* PHP version: 7.4

### PR DESCRIPTION
**Pull request by mparvazi**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.8.x
| Description?      | Selecting a time-val button (day, month, year,...), doesn't change the calendar date-range and causes a jquery error. [admin-dev/themes/default/js/calendar.js]
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | #25966
| How to test?      | Select a time-val button to change the calendar date-range.
| Possible impacts? | BO > Dashboard

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/25986)

### CODE REVIEW

**Comment by Progi1984** [comment]
@mparvazi Could you target the `1.7.8.x` branch for embedding this PR in next patch version ?

File: admin-dev/themes/default/js/calendar.js [hunk 1]
```diff
@@ -309,7 +309,7 @@ function setPreviousYear() {
 
 $(document).ready(() => {
   // Instanciate datepickers
-  const datepickerStart = $('.datepicker1').daterangepicker({
+  datepickerStart = $('.datepicker1').daterangepicker({
```
Comment (Progi1984) [inline admin-dev/themes/default/js/calendar.js:312]:
```suggestion
  let datepickerStart = $('.datepicker1').daterangepicker({
```

File: admin-dev/themes/default/js/calendar.js [hunk 2]
```diff
@@ -320,7 +320,7 @@ $(document).ready(() => {
     }
   }).data('daterangepicker');
 
-  const datepickerEnd = $('.datepicker2').daterangepicker({
+  datepickerEnd = $('.datepicker2').daterangepicker({
```
Comment (Progi1984) [inline admin-dev/themes/default/js/calendar.js:323]:
```suggestion
  let datepickerEnd = $('.datepicker2').daterangepicker({
```

Comment (NeOMakinG) [reply admin-dev/themes/default/js/calendar.js:323]: @Progi1984 as datepickerEnd is already declared below, you don't need to declare it again as it's inside the same scope

**Review by PierreRambaud — CHANGES_REQUESTED** [review]
Could you please rebase your pull request, we won't accept merge commits:
image

**Review by sowbiba — COMMENTED** [review]
Viewed. Waiting for fixes

**Comment by mparvazi** [comment]
Hello @PierreRambaud,
I'm newbie to git and I use Web UI version of git. If you don't mind, please tell me how to rebase my PR.
Thanks.

**Comment by sowbiba** [comment]
🤔  1.7.8.x or develop

I think you target the wrong branch

**Review by sowbiba — COMMENTED** [review]
🤔  1.7.8.x or develop ?

I think you target the wrong branch

**Comment by matks** [comment]
I'm really sorry @mparvazi but there's an issue, once again 😅 
Capture d’écran 2021-10-05 à 18 11 46

**Comment by mparvazi** [comment]
@matks It seems I've made lot of mistakes in this PR. Is it better to close this PR and make a new request?
Sorry, I'm newbie at Git and need your help. Please tell me what should I do.
Thanks.

**Comment by matks** [comment]
> @matks It seems I've made lot of mistakes in this PR. Is it better to close this PR and make a new request?*

I think yes the easy way to fix this is to redo a PR. Make sure to use `1.7.8.x` as base branch for your new PR when you create it, not `develop`.

I tried to fix your branch myself but there's too much mix in it 😅

**Comment by mparvazi** [comment]
> I tried to fix your branch myself but there's too much mix in it 😅

@matks Ok, I got it 😁

**Comment by mparvazi** [comment]
PR needs to redo!

**Pull request closed without merging**